### DISCUSSION TITLE
ObeyGDBrowser 1.0 (The biggest and best update)

### DISCUSSION THREAD

**Discussion by MigMatos** [MigMatos/ObeyGDBrowser#13] [Announcements]
🛠Bug fixes:

- Fixed filters in Legacy/nginx servers.
- Fixed songID searchs.
- Fixed visuals errors from "loadings".
- Fixed bad redirects from profiles.
- Fixed issues with Firefox browser.
- Fixed search no working with "starred" tab correctly.

🔥New Features:

- Songs Tab (search songs, artists and more!)

- Added custom lists in Search

- Added MapPacks 

- Added Gauntlets 

- Modern UI in Profiles, Song, Login & GDPS Settings

- Added Mod Actions
- - You can rate levels
- - Add, Edit or Delete MapPacks!
- - Add, Edit or Delete Gauntlets!

- Added super dynamic searchs!

- New login system

- Link your Discord Account with your GDPS via ObeyGDBrowser!

- Modify your social links via ObeyGDBrowser!

- NEW UPDATER!!!! (my better updater LOL!)

- MODERATOR TAB???? Available October 6th 🔥🔥

❗HEY IMPORTANT:

- If you get "500" errors when updating, simply refresh the page again.
- If you have errors installing, use the old installer ( `old_installer.php` ) [New users].
-  **If your update file got corrupted, check that tutorial:** https://www.youtube.com/watch?v=h9NnAlJxy1o

**Video of how install ObeyGDBrowser:**
https://www.youtube.com/watch?v=Rl0w0EiS7uY

## You need help? Join to Discord Support!

[![OGDTeam Invite](https://invidget.switchblade.xyz/EbYKSHh95B)](https://discord.gg/EbYKSHh95B)

## What's Changed
* api gauntlets, me mato by @gdnoxi in https://github.com/MigMatos/ObeyGDBrowser/pull/6
* better gdps settings css by @gdnoxi in https://github.com/MigMatos/ObeyGDBrowser/pull/9
* Add gauntlet **background** colors by @YeahhColix in https://github.com/MigMatos/ObeyGDBrowser/pull/11
* Add missing gauntlets & text colors by @YeahhColix in https://github.com/MigMatos/ObeyGDBrowser/pull/12

## New Contributors
* @gdnoxi made their first contribution in https://github.com/MigMatos/ObeyGDBrowser/pull/6

**Full Changelog**: https://github.com/MigMatos/ObeyGDBrowser/compare/0.27.1.1...1.0-stable

<hr /><em>This discussion was created from the release <a href='https://github.com/MigMatos/ObeyGDBrowser/releases/tag/1.0-stable'>ObeyGDBrowser 1.0 (The biggest and best update)</a>.</em>

**Comment by Minhsanggd** [comment]
IT'S NOT WORKING:(

**Comment by flessan** [comment]
Um.. Bugs On incl/Lib.. Reinstall And it works 🐱 It's okay just commenting
